TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/base/managers/BaseInstanceManager.java

```diff
@@ -459,6 +471,31 @@ private void blockPort(int port) {
         recentlyUsedPorts.put(port, 60);
     }
 
+    private int getScreenVersion() {
+        try {
+            Process getversion = new ProcessBuilder("/bin/sh", "-c", "screen -v").start();
+            StringBuilder textBuilder = new StringBuilder();
+            try (Reader reader = new BufferedReader(new InputStreamReader
+                    (getversion.getInputStream(), Charset.forName(StandardCharsets.UTF_8.name())))) {
+                int c = 0;
+                while ((c = reader.read()) != -1) {
+                    textBuilder.append((char) c);
+                }
+            }
+            String[] log = textBuilder.toString().split(" ");
+            if (log.length > 2) {
+                String version = log[2].replace(".", "");
+                try {
+                    return Integer.parseInt(version);
+                } catch (NumberFormatException ignored) {
+                }
+            }
+        } catch (Exception exception) {
+
+        }
+        return 0;
```

Comment:
Return a high value if output is not readable because that should only happen in future versions.

Log a warning message